### PR TITLE
[zh] synchronize translate service-catalog.md

### PR DESCRIPTION
**Pull request by shuaijinchao**
/language zh

### CODE REVIEW

File: content/zh/docs/concepts/extend-kubernetes/service-catalog.md [hunk 1]
```diff
@@ -66,10 +66,7 @@ It is implemented as an extension API server and a controller, using etcd for st
 与服务代理进行通信，并作为 Kubernetes API 服务器的中介，以便协商启动部署和获取
 应用程序使用托管服务时必须的凭据。
 
-服务目录实现为一个扩展 API 服务器和一个控制器，使用 Etcd 提供存储。
-它还使用了 Kubernetes 1.7 之后版本中提供的
-[聚合层](/zh/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/)
-来呈现其 API。
+它是 [基于 CRDs](/zh/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources) 架构实现的。
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/service-catalog.md:69]:
```suggestion
它是[基于 CRDs](/zh/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources)
架构实现的。
```

Comment (shuaijinchao) [reply content/zh/docs/concepts/extend-kubernetes/service-catalog.md]: updated

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**